src/frontend/src/lib/constants/transition.constants.ts

```diff
@@ -0,0 +1,11 @@
+import { quintOut } from 'svelte/easing';
+import type { SlideParams } from 'svelte/transition';
+
+export const SLIDE_DURATION = { duration: 250 };
+
+export const SLIDE_QUINT_OUT: Pick<SlideParams, 'easing' | 'axis'> = {
```

Comment:
undecided between `SLIDE_QUINT_OUT` and `SLIDE_EASING`

Reply:
No strong opinion